Fitness/Views/Cells/HabitTrackerOnboardingCell.swift

```diff
@@ -152,7 +170,7 @@ class HabitTrackerOnboardingCell: UITableViewCell {
         }
         
         titleLabel.snp.updateConstraints { make in
-            make.trailing.equalToSuperview().offset(-132)
+            make.trailing.equalToSuperview().offset(-145)
```

Comment:
Pull these to top of file! Also when using equalToSuperview() you should try and use `.inset(x)`